### PR TITLE
[MRG] Replace hard-coded helm version with environment variable in deploy.py

### PR DESCRIPTION
**Pull request by sgibson91**
This PR will replace the hard-coded helm version in deploy.py with an environment variable that can be set within .travis.yml. This will mean that when the helm version is bumped, it only needs to be updated in one place.

TODOS:

- [x] Update deploy.py to include a HELM_VERSION environment variable
- [x] Update .travis.yml to extract the helm version and parse to deploy.py

### CODE REVIEW

File: deploy.py [hunk 1]
```diff
@@ -15,6 +15,10 @@
 HERE = os.path.dirname(__file__)
 ABSOLUTE_HERE = os.path.dirname(os.path.realpath(__file__))
 
+# Get helm version environment variable
+HELM_VERSION = os.get_env("HELM_VERSION", None)
+if HELM_VERSION is None:
+    raise Exception("HELM_VERSION environment version must be set")
```
Comment (sgibson91) [inline deploy.py]:
```suggestion
    raise Exception("HELM_VERSION environment variable must be set")
```

File: .travis.yml [hunk 2]
```diff
@@ -61,6 +61,8 @@ before_deploy:
 - |
   # Stage 2, Step 2: Set up helm!
   helm init --client-only
+  TRAVIS_HELM_VERSION=`helm version -c --short | sed 's/.*v\([0-9\.]*\).*/\1/'`
+  echo ${TRAVIS_HELM_VERSION}
```
Comment (minrk) [inline .travis.yml]: Can we use $HELM_VERSION to set the version in the the "Install helm" step? on line 45 above? That way we don't need to query it here.

Comment (sgibson91) [reply .travis.yml]: Done in 0356912 ✨

Comment (minrk) [reply .travis.yml]: Thanks!

**Pull request merged**